package.json

```diff
@@ -190,7 +190,6 @@
 		"react-player": "^2.9.0",
 		"rehype-sanitize": "^5.0.1",
 		"remark-gfm": "^3.0.1",
-		"rivet-graphql": "^0.3.1",
```

Comment:
I've removed `rivet-graphql` - it was used for the `io` sites, but is no longer needed. This may seem unrelated to the Sentinel content deploy preview issue we're trying to target, but this removal seemed necessary to make the `next` version alignment work as expected.